src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2401,7 +2403,10 @@ def _update_addons(cmd, instance, subscription_id, resource_group_name, addons,
             addon_profiles[addon] = addon_profile
         else:
             if addon not in addon_profiles:
-                raise CLIError("The addon {} is not installed.".format(addon))
+                if addon == 'kubeDashboard':
```

Comment:
case insenstivie compare?

Reply:
It's fine here. `if addon not in addon_profiles` guarantees 'kubeDashboard' is the only possible case.